src/js/components/controls/NumericSlider.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import '../../../scss/components/NumericSlider.scss'
+
+class NumericSlider extends Component{
+    constructor(props){
+        super(props);
+        this.props = props;
+    }
+
+    render() {
+        return [<input id={this.props.setting.key} name={this.props.setting.key}
```

Comment:
You don't need to wrap the return value with brackets if there's only one element.